src/main/kotlin/baseball/view/OutputGuideView.kt

```diff
@@ -0,0 +1,16 @@
+package baseball.view
+
+import baseball.config.GameMsg.GAME_OVER
+import baseball.config.GameMsg.START_GAME
+
+class OutputGuideView {
+    fun showStartMsg() {
+        println(START_GAME)
+    }
```

Comment:
> Prefer using an expression body for functions with the body consisting of a single expression.

한 줄로 된 함수는 expression body를 사용하는 것이 권장 된다고 하네요.
https://kotlinlang.org/docs/coding-conventions.html#functions


```suggestion
    fun showStartMsg() = println(START_GAME)
```

Reply:
그러네요. 여기서는 표현식으로 수정하는게 더 좋아보이네요